misc/plot-ampliconstats

```diff
@@ -0,0 +1,884 @@
+#!/usr/bin/env perl
+#
+#    Copyright (C) 2020 Genome Research Ltd.
+#
+#    Author: James Bonfield <jkb@sanger.ac.uk>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+# Note: this script requires gnuplot version 5.0 or later.
+
+use strict;
+use Getopt::Long;
+
+my $version = "1.0";
+
+#-----------------------------------------------------------------------------
+# Command line argument handling & help
+
+my %opts;
+
+my $res = GetOptions("size=s"      => \$opts{size},
+                     "size2=s"     => \$opts{size2},
+                     "size3=s"     => \$opts{size3},
+                     "help"        => \$opts{help},
+                     "page=i"      => \$opts{page},
+		     "amp_add=i"   => \$opts{amp_add},
```

Comment:
It looks like there are some tabs here.  It looks like the anti-tab system missed them due to no `.pl` on the file name.